tests/test_nfl_validation.py

```
@@ -1,12 +1,14 @@
 import sys
 import pathlib
+
 sys.path.insert(0, str(pathlib.Path(__file__).resolve().parents[1]))
 from nfl.parser import parse_nfl
 from jsonschema import Draft202012Validator
 import json
 
-SCHEMA = json.load(open('schema/nfl.schema.json'))
-NFL_FILES = pathlib.Path('.').glob('**/*.nfl')
+SCHEMA = json.load(open("schema/nfl.schema.json"))
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)